src/main/target/common_post.h

```diff
@@ -186,12 +186,8 @@
 #endif
 #endif
 
-// Add VARIO if BARO or GPS is defined. Remove when none defined.
-#if defined(USE_BARO) || defined(USE_GPS)
-#ifndef USE_VARIO
-#define USE_VARIO
-#endif
-#else
+// Remove USE_VARIO if BARO or GPS is not defined.
+#if (!defined(USE_BARO) || !defined(USE_GPS)) && defined(USE_VARIO)
```

Comment:
We want to support the user's request to include vario when they additionally include either GPS or BARO, but if they provide neither, then we don't go ahead.
Maybe this should be:
```
// Remove USE_VARIO if neither BARO nor GPS are defined.
#if defined(USE_VARIO) && !(defined(USE_BARO) || defined(USE_GPS))
#undef USE_VARIO
#endif
```

Reply:
Sorted.

Reply:
@ctzsnooze but now you need to include VARIO manually. You don't get it automatically anymore if the requirements for its use are met. Is this wanted? Seems wrong to me.